docs/components/customization/user-profile.mdx

```diff
@@ -1,113 +1,31 @@
 ---
-title: <UserProfile /> customization
+title: Add custom pages and links to the <UserProfile /> component
 description: Learn how to add custom pages and include external links within the navigation sidebar of the <UserProfile /> component.
 ---
 
-# `<UserProfile />` customization
+# Add custom pages and links to the `<UserProfile />` component
 
-The [`<UserProfile />`][userprofile-ref] component supports the addition of custom pages and use of external links in the navigation sidebar. 
+The [`<UserProfile />`](/docs/components/user/user-profile) component supports the addition of custom pages and use of external links in the navigation sidebar.
 
-## `<UserProfile.Page />`
+There are two ways to render the `<UserProfile />` component:
+- As a modal that opens when the [`<UserButton />`](/docs/components/user/user-button) is clicked
+- As a dedicated page
 
-Custom pages can be rendered inside the [`<UserProfile />`][userprofile-ref] component and provide a way to incorporate app-specific settings or additional functionality.
+This guide includes examples for both use cases. You can select the tabs on the code examples to see the implementation for your preferred use case.
 
-To add a custom page to the [`<UserProfile />`][userprofile-ref] component, use the `<UserProfile.Page />` component.
+By default, the `<UserButton />` sets `userProfileMode='modal'`. If you are using the default settings, then you should use the `<UserButton />` examples. However, if you are using the  `userProfileMode='navigation'` and `userProfileUrl='/user-profile'` props on the `<UserButton />` component, then you should use the `Dedicated page` examples.
```

Comment:
Do we like this better??

```suggestion
This guide includes examples for both use cases. You can select one of the following two tabs on the code examples to see the implementation for your preferred use case:

- `<UserButton />` tab: By default, the `<UserButton />` sets `userProfileMode='modal'`. If you are using the default settings, then you should select this tab.
- `Dedicated page` tab: If you do not want the `<UserProfile />` to open as a modal, then you should select this tab. For these examples, you need to set `userProfileMode='navigation'` and `userProfileUrl='/user-profile'` on the `<UserButton />` component.
```

Reply:
Yeah, I think this is a little easier to read